src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  const result = (a) => {
```

Comment:
I must say, you can just return here, instead of defining it and returning

```js
  return (a) => {
    count++;
    return (b) => {

Reply:
@UmizDemud is totally right here 👍